packages/core/src/queries/sso-connectors.ts

```diff
@@ -92,4 +93,12 @@ export default class SsoConnectorQueries extends SchemaQueries<
       throw new DeletionError(SsoConnectorIdpInitiatedAuthConfigs.table);
     }
   }
+
+  async findIdpInitiatedSamlSsoSessionById(id: string) {
+    const { table, fields } = convertToIdentifiers(IdpInitiatedSamlSsoSessions);
+    return this.pool.maybeOne<IdpInitiatedSamlSsoSession>(sql`
```

Comment:
nit: use `buildFindEntityByIdWithPool`

Reply:
`buildFindEntityByIdWithPool` uses `pool.one`, which will throw an `entity.not_exists` error.  Would like to safely fetch the session here.